File: RealEstateNew.Domain/Entities/Item.cs
namespace RealEstateNew.Domain.Entities
{
    public class Item:BaseEntity
    {
        public int AdvertiseNo { get; set; }
        public int CategoryId { get; set; }
        public Category? Category { get; set; }
        public int CityId { get; set; }
        public City? City { get; set; }
        public int DistrictId { get; set; }
        public District? District { get; set; }
        public int PropertyTypeId { get; set; }
        public PropertyType? PropertyType { get; set; }

        public double Latitude { get; set; }
        public double Longitude { get; set; }
        public double Soum { get; set; }
        public double Limit { get; set; }
        public double StreetWidth { get; set; }
        public double Space { get; set; }
        public double PricePerMeter { get; set; }
        public string? Description { get; set; }
        public string? HashedPassword { get; set; }
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

        public List<Image> Images { get; set; } = new();
    }
}

File: NewRealEstate.Front/src/app/core/models/item.ts
import { Booking } from "./booking";
import { Category } from "./category";
import { City } from "./city";
import { District } from "./district";

export interface Item {
    id: number;
    name?: string;
    image?: string;
    status?: boolean;
    advertiseNo: number;

    categoryId: number;
    category?: Category;

    cityId: number;
    city?: City;

    districtId: number;
    district?: District;

    propertyTypeId: number;
    //propertyType?: PropertyType;

    latitude: number;
    longitude: number;
    soum: number;
    limit: number;
    streetWidth: number;
    space: number;
    pricePerMeter: number;

    description?: string;
    hashedPassword?: string;
    createdAt: Date;

    images: Image[];
    bookings?: Booking[];
}

export interface Image {
    id: number;
    imageUrl?: string;
    status: boolean
}
